Pear/Controller/MessagesViewController.swift

```diff
@@ -0,0 +1,191 @@
+
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private let subtitleLabel = UILabel()
+    private var messagesTableView = UITableView()
+
+    // MARK: - Private Data Vars
+    private let databaseRef = Database.database().reference()
+    private var messageUsers: [MessageUser] = []
+    private var user: User
+    var timer: Timer?
+
+    init(user: User) {
+        self.user = user
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .backgroundLightGreen
+        title = "Past pears"
+
+        navigationController?.navigationBar.isHidden = false
+        navigationController?.navigationBar.barTintColor = .backgroundLightGreen
+        navigationController?.navigationBar.shadowImage = UIImage()
+        navigationController?.navigationBar.titleTextAttributes = [
+            .font: UIFont.getFont(.medium, size: 24)
+        ]
+
+        backButton.setImage(UIImage(named: "backArrow"), for: .normal)
+        backButton.snp.makeConstraints { make in
+            make.size.equalTo(Constants.Onboarding.backButtonSize)
+        }
+        backButton.addTarget(self, action: #selector(backPressed), for: .touchUpInside)
+        navigationItem.leftBarButtonItem = UIBarButtonItem(customView: backButton)
+
+        subtitleLabel.text = "Keep in touch with each other :)"
+        subtitleLabel.textColor = .greenGray
+        subtitleLabel.font = ._12CircularStdBook
+        view.addSubview(subtitleLabel)
+
+        messagesTableView.register(MessagesTableViewCell.self, forCellReuseIdentifier: MessagesTableViewCell.reuseId)
+        messagesTableView.backgroundColor = .backgroundLightGreen
+        messagesTableView.separatorStyle = .none
+        messagesTableView.dataSource = self
+        messagesTableView.delegate = self
+        view.addSubview(messagesTableView)
+
+        getUserMessages()
+        setupConstraints()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        navigationController?.interactivePopGestureRecognizer?.delegate = self
+    }
+
+    @objc private func backPressed() {
+        navigationController?.popViewController(animated: true)
+    }
+
+    private func setupConstraints() {
+        subtitleLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(10)
+        }
+        messagesTableView.snp.makeConstraints { make in
+            make.top.equalTo(subtitleLabel.snp.bottom).offset(20)
+            make.leading.trailing.bottom.equalToSuperview()
+        }
+    }
+
+    private func getUserMessages() {
+        getMessageMatches(netId: user.netID) { matches in
+            matches.forEach { match in
```

Comment:
`for match in matches {`. Off the top of my head, one of the few times to use `forEach` is when you're concisely passing in a named function/closure. (e.g. `matches.forEach(doSomething)`)